scripts/run.sh

```diff
@@ -12,29 +12,31 @@ then
     export WIDTH=32
     export HEIGHT=16
 else
-    export WIDTH=7
-    export HEIGHT=6
+    export WIDTH=12
+    export HEIGHT=12
 fi
 
 # build CGRA
 make garnet_verilog
 
 # build apps
-make pointwise
-make absolute
+if [ -n "$BUILDKITE" ] && [ -z "$PR" ];
```

Comment:
Can we move this to the button? I want to fail fast on the small tests first.

Reply:
Moved the larger apps to the bottom of the flow.